### PR TITLE
full size FanArts

### LINKED ISSUE #32: link to full size FanArts
on clicking a FanArt in the Home Page, the FanArt full size Image to be opened.
currently all 6 of them links to the Home page.

### PR DESCRIPTION
**Pull request by raajtilaksarma**
On clicking a FanArt in the Home Page, the FanArt full size image is opened.

Fixes #32

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -163,13 +163,13 @@ <h3>Pratyasha Kalita <br><span class="position">Artist</span></h3>
                     </a>
                   </div>
 
-                </div>
+                </div>`
```
Comment (Shankusu993) [inline index.html]: can you just correct this, I guess it was added by mistake.

**Review by Shankusu993 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (raajtilaksarma) [reply index.html]: on it

**Review by raajtilaksarma — COMMENTED** [review]
typo corrected

**Comment by Shankusu993** [comment]
LGTM, nice job. I'll merge  it.

**Pull request merged**